drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -462,12 +472,7 @@ void RasterizerGLES3::_blit_render_target_to_screen(RID p_render_target, Display
 // is this p_screen useless in a multi window environment?
 void RasterizerGLES3::blit_render_targets_to_screen(DisplayServer::WindowID p_screen, const BlitToScreen *p_render_targets, int p_amount) {
 	for (int i = 0; i < p_amount; i++) {
-		const BlitToScreen &blit = p_render_targets[i];
-
-		RID rid_rt = blit.render_target;
-
-		Rect2 dst_rect = blit.dst_rect;
-		_blit_render_target_to_screen(rid_rt, p_screen, dst_rect, blit.multi_view.use_layer ? blit.multi_view.layer : 0, i == 0);
+		_blit_render_target_to_screen(p_screen, p_render_targets[i], i == 0);
```

Comment:
Figured just providing our data as it already is as a parameter, would be much more efficient than copying all the members as parameters. 

I wonder if we should insert `_blit_render_target_to_screen` into `blit_render_targets_to_screen`, there really isn't a good benefit to these being split.